src/avrdude.1

```diff
@@ -1257,9 +1257,12 @@ on a non-zero verbosity level the line numbers are printed, too.
 .It Ar sig
 Display the device signature bytes.
 .It Ar part
-Display the current part settings and parameters.  Includes chip
-specific information including all memories supported by the
-device, read/write timing, etc.
+Display the current part information. Includes chip specific information
```

Comment:
". Includes chip specific information" could be removed for brevity (same in .texi)